sdk/python/feast/repo_config.py

```diff
@@ -24,6 +24,7 @@
     "datastore": "feast.infra.online_stores.datastore.DatastoreOnlineStore",
     "redis": "feast.infra.online_stores.redis.RedisOnlineStore",
     "dynamodb": "feast.infra.online_stores.dynamodb.DynamoDBOnlineStore",
+    "astra": "feast.infra.online_stores.astra.AstraDBOnlineStore",
```

Comment:
This is fine for initial testing. We probably don't want to merge this file in quite yet and instead have a pointer to the plugin repo.

https://github.com/feast-dev/feast/issues/1892 shows our plans to eventually make these tests pluggable for plugin repos.

Reply:
Should I create in plugin format?